.travis.yml

```diff
@@ -5,6 +5,10 @@ node_js:
   - "6"
   - "7"
   - "8"
+matrix:
+  allow_failures:
+  - node_js: 6
+  - node_js: 7
```

Comment:
@simison any ideas here? I haven't used `allow_failures` before.

Reply:
Looks good otherwise but versions above are strings `"6"` and in matrix they're integers, might be it fails because of that.

See e.g. https://github.com/meanjs/mean/blob/482c38c287ff31688f0c9efa4ad4331009beeb40/.travis.yml#L4-L11

Reply:
`fast_finish: true` is handy btw: https://blog.travis-ci.com/2013-11-27-fast-finishing-builds/